akka-http-backend/src/main/scala/com/softwaremill/sttp/akkahttp/AkkaHttpBackend.scala

```diff
@@ -358,14 +385,15 @@ object AkkaHttpBackend {
       customHttpsContext: Option[HttpsConnectionContext] = None,
       customConnectionPoolSettings: Option[ConnectionPoolSettings] = None,
       customLog: Option[LoggingAdapter] = None
+  )(
+      http: AkkaHttpClient = AkkaHttpClient.fromAkkaHttpExt(actorSystem, customHttpsContext, customLog)
```

Comment:
Maybe we should just use `fromAkkaHttpExt(actorSystem, None, None)` here and not take parameters for `customHttpsContext` and `customLog`? If somebody wants to use custom ones they can provide an AkkaHttpClient with changed parameters themselves.

Reply:
I would prefer not to change the API, or at least keep it backwards-(source)-compatible. This change might wait until sttp 2 (when it comes ;) ). So I would keep the signatures of existing methods intact, and add a new variant which takes an `AkkaHttpClient`. The others might delegate to that particular variant.

Reply:
Can we make `make` public then? Otherwise we'll need to duplicate both variants (the one that takes an ActorSystem and one that creates it itself).